pcilib.man

```diff
@@ -154,6 +174,12 @@ Path to the sysfs device tree.
 .B devmem.path
 Path to the /dev/mem device.
 .TP
+.B acpimcfg.path
+Path to the ACPI MCFG table.
+.TP
+.B efisystab.path
+Path to the EFI system table.
+.TP
```

Comment:
Technically EFI system table has nothing with ECAM. It can be used also for other purposes or other backends, so I put it into `<name>.path` naming convention like there are other files. I have just used EFI system table as a source for ACPI which is source of the MCFG table, which is one option how to retrieve ECAM mapping.

Reply:
The naming convention works differently: in `a.b`, `a` is a module (e.g., a back end) and `b` the module's parameter. But there is no module `efisystab` nor `acpimcfg`.

You already broke that convention when introducing `devmem.path` in `mmio-ports`. Not only the name is not systematic, but the parameter is also registered twice with the same name. It is however my fault that I haven't spotted the problems during review.

Even though technically the EFI systab path could be used by multiple modules, they do not exist and we have no idea of how they would look like. So what is really just settings of the ECAM module is scattered over the whole namespace, unnecessarily confusing users.

I think that the best solution is moving all of that to the `ecam.*` subtree and should there ever arise multiple modules reading ACPI/EFI tables, they will likely share some common infrastructure, which will have its own subtree of option names (with previously used `ecam.*` names working as backward-compatible aliases).

Reply:
I renamed them.